DisplaySettings/CHANGELOG.md

```diff
@@ -15,6 +15,25 @@ All notable changes to this RDK Service will be documented in this file.
 * Changes in CHANGELOG should be updated when commits are added to the main or release branches. There should be one CHANGELOG entry per JIRA Ticket. This is not enforced on sprint branches since there could be multiple changes for the same JIRA ticket during development. 
 
 * For more details, refer to [versioning](https://github.com/rdkcentral/rdkservices#versioning) section under Main README.
+## [1.0.19] - 2023-03-07
```

Comment:
Pls make sure that we are only updating changelog for the change that u are bringing. we shouldn't be syncing the changelog with sprint